sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1963,3 +1963,64 @@ case class Epoch(child: Expression, timeZoneId: Option[String] = None)
     defineCodeGen(ctx, ev, c => s"$dtu.getEpoch($c, $zid)")
   }
 }
+
+@ExpressionDescription(
+  usage = "_FUNC_(field, source) - Extracts a part of the date/timestamp.",
+  arguments = """
+    Arguments:
+      * field - selects which part of the source should be extracted. Supported string values are:
+                ["MILLENNIUM", "CENTURY", "DECADE", "YEAR", "QUARTER", "MONTH",
+                 "WEEK", "DAY", "DAYOFWEEK", "DOW", "ISODOW", "DOY",
+                 "HOUR", "MINUTE", "SECOND"]
```

Comment:
We don't need to list up all the alternatives somewhere for users?

Reply:
I documented all values for consistency.